.github/workflows/development.yml

```diff
@@ -30,15 +30,15 @@ jobs:
 
       - name: black
         run: |
-          black --check setup.py xain_fl
+          black --line-length 100 --check setup.py xain_fl
```

Comment:
Why not using the default 88 length?

Reply:
because the team agreed upon using black with 100 characters line length after the team merger (i know you haven't been around at that time, no offense, our style guide can be found here: https://xainag.atlassian.net/wiki/spaces/FP/pages/208928820/Python+Code+Style+Guide)

Reply:
therefore, the more important question is, why nobody took care of this change to the CI before...